app/vep/models/client_model.py

```diff
@@ -1 +1,38 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+class FieldOptions:
+    GENCODE_LABEL: str = "GENCODE"
+    GENCODE_VALUE: str = "gencode_comprehensive"
+
+class Options(BaseModel):
+  label: str = FieldOptions.GENCODE_LABEL
+  value: str = FieldOptions.GENCODE_VALUE
+
+class TranscriptSet(BaseModel):
+    label: str
+    description: str = None
+    type: str = "select"
+    options: List[Options] = [Options()]
+    default_value: str = FieldOptions.GENCODE_VALUE
+
+class Symbol(BaseModel):
+    label: str = "Gene symbol"
+    description: str = None
```

Comment:
str | None

Reply:
There should be a class like `BooleanOption` (or `CheckboxOption`) which will describe the type. Both symbol and biotype are instances of that class; they do not need their own classes.